android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboardDownloaderActivity.java

```diff
@@ -413,13 +408,8 @@ public static boolean isCustom(String u) {
       String fontSourceString;
       try {
         fontSourceString = jsonFont.getString(KMManager.KMKey_FontSource);
-        if (fontSourceString.endsWith(".ttf") || fontSourceString.endsWith(".otf")) {
-          urls.add(baseUri + fontSourceString);
-        } else if (isOskFont && (fontSourceString.endsWith(".svg") || fontSourceString.endsWith(".woff"))) {
+        if (FileUtils.hasFontExtension(fontSourceString)) {
```

Comment:
We've lost support for WOFF and SVG osk fonts with this change. That could break support for some older packages on some older Android devices.